docker.local/bin/conductor/register_authorizer.sh

```diff
@@ -10,6 +10,7 @@ printf '{"client_id":"1746b06bb09f55ee01b33b5e2e055d6cc7a900cb57c0a3a5eaabb8a0e7
 
 ./zwalletcli/zwallet getbalance --wallet wallet.json
 
+./zwalletcli/zwallet send --tokens 10 --desc 'to auth 1' --to_client_id 1746b06bb09f55ee01b33b5e2e055d6cc7a900cb57c0a3a5eaabb8a0e7745802
```

Comment:
You will get `invalid_request: Invalid request (from and to client should be different)"}` error if you transfer token to this address, which is the wallet itself.